how-to-guides/how-it-works/notebooks/Neptune_API_Tour.ipynb

```diff
@@ -93,38 +93,42 @@
    "source": [
     "import neptune.new as neptune\n",
     "\n",
-    "run = neptune.init(project=\"common/quickstarts\", api_token=\"ANONYMOUS\")"
+    "run = neptune.init_run(project=\"common/quickstarts\", api_token=neptune.ANONYMOUS_API_TOKEN)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "You tell Neptune: \n",
+    "Click on the link above to open this run in Neptune. For now it is empty, but keep the tab with run open to see what happens next.\n",
     "\n",
-    "* **who you are**: your Neptune API token `api_token` \n",
-    "* **where you want to send your data**: your Neptune `project`.\n",
+    "With the `api_token` and `project` arguments, you tell Neptune: \n",
     "\n",
-    "At this point you have new run in Neptune. For now on you will use `run` to log metadata to it.\n",
+    "* **who you are**: with your Neptune API token\n",
+    "* **where you want to send your data**: your Neptune project\n",
+    "\n",
+    "At this point you have new run in Neptune. From now on, we'll use the `run` object to log metadata.\n",
     "\n",
     "---\n",
     "\n",
     "**Note**\n",
     "\n",
     "\n",
-    "Instead of logging data to the public project 'common/quickstarts' as an anonymous user 'neptuner' you can log it to your own project.\n",
+    "Instead of logging data to the public project `'common/fbprophet-integration'` as an anonymous user, you can log it to your own project.\n",
     "\n",
-    "To do that:\n",
+    "To find and copy your credentials:\n",
     "\n",
-    "1. Get your [Neptune API token](https://docs.neptune.ai/getting-started/installation#authentication-neptune-api-token)\n",
-    "2. Pass the token to ``api_token`` argument of ``neptune.init()`` method: ``api_token=YOUR_API_TOKEN``\n",
-    "3. Pass your project to the ``project`` argument of ``neptune.init()``.\n",
+    "1. In the Neptune app, click your avatar and select **Get your API token**.\n",
```

Comment:
Please add a link to the docs:
https://docs.neptune.ai/setup/setting_api_token/

Reply:
This is as per @normandy7's notebook template

Reply:
Since this is the api-tour and might be one of the first pieces of Neptune code a user runs, I think it would make sense to actually send them over to the docs to explain how to fetch and use the API token. Then they can decide if they want to use an env variable or getpass

Reply:
Fair enough, we can add a link. The template proposes `getpass()` instead of passing the token directly to `api_token`, though. Why don't we use that method here?

Reply:
@normandy7 - updated. please have a look